overlord/snapstate/snapstate_test.go

```diff
@@ -5046,6 +5082,9 @@ func (s *snapmgrTestSuite) TestTryUndoRemovesTryFlagLeavesJailMode(c *C) {
 	s.testTrySetsTryMode(snapstate.Flags{JailMode: true}, c)
 }
 func (s *snapmgrTestSuite) TestTryUndoRemovesTryFlagLeavesClassic(c *C) {
+	if !dirs.SupportsClassicConfinement() {
```

Comment:
Slightly strange that `SupportsClassicConfinement()` is a member of `dirs` - I understand why but still feels wrong. Anyway, not something to be concerned about for this branch.